src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -31,28 +33,133 @@ def setUp(self):
   def tearDown(self):
     pass
 
+  def triggerPpaCreationAndWaitUntilComplete(self, ppa_creation_request):
```

Comment:
Isn't this already defined somewhere else? Prefer to put the code in a common location rather than to duplicate it.

Reply:
two   methods are placed in "sas_testcase.py" file